sql/handler.cc

```diff
@@ -2396,6 +2396,14 @@ int ha_prepare_low(THD *thd, bool all) {
       const bool run_wsrep_hooks = wsrep_run_commit_hook(thd, all);
       int err;
 
+      auto rpl_com = get_commit_order_manager(thd);
+      if (rpl_com != nullptr) {
+        if (rpl_com->wait_on_graph((Slave_worker*)thd->rli_slave)) {
```

Comment:
Can we use `dynamic_cast` here?
```suggestion
        if (rpl_com->wait_on_graph(dynamic_cast<Slave_worker *>(thd->rli_slave)) {
```